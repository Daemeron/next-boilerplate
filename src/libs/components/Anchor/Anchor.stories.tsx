import React, { Fragment } from 'react';

import { Anchor } from './Anchor';

export default {
  title: 'Components/Anchor',
};

export const WithText = (): JSX.Element => (
  <Fragment>
    <Anchor>dummy text</Anchor>
  </Fragment>
);
