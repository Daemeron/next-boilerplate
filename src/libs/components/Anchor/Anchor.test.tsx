import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import { Anchor } from './Anchor';

describe('Link', () => {
  test('matches snapshot', () => {
    const anchor = render(<Anchor>test</Anchor>);
    expect(anchor.container).toMatchSnapshot();
  });
});
