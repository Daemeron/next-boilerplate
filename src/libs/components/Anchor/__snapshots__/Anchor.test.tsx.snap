// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link matches snapshot 1`] = `
<div>
  <a
    class=""
  >
    test
  </a>
</div>
`;
